consensus/module.go

```diff
@@ -64,12 +65,19 @@ type consensusModule struct {
 	paceMaker         pacemaker.Pacemaker
 	leaderElectionMod leader_election.LeaderElectionModule
 
-	logger    *modules.Logger
-	logPrefix string
+	logger *modules.Logger
 
 	stateSync state_sync.StateSyncModule
 
 	hotstuffMempool map[typesCons.HotstuffStep]*hotstuffFIFOMempool
+
+	// block responses received from peers are collected in this channel
+	blocksReceived chan *coreTypes.Block
```

Comment:
I left a more detail comment about it below, but this should be block responses, not `blocks` directly.